Dockerfile

```diff
@@ -71,7 +71,7 @@ COPY docker/start.sh /start.sh
 
 WORKDIR /etc/temporal
 
-ENV SERVICES="--services=history --services=matching --services=frontend --services=worker"
+ENV SERVICES="history:matching:frontend:worker"
```

Comment:
Semicolon is standard way to separate array items concatenated in one string (i.e. `$PATH`).

Reply:
Comma is still supported though.